frontend/pages/ManageControlsPage/OSSettings/cards/DiskEncryption/DiskEncryption.tsx

```diff
@@ -128,7 +128,7 @@ const DiskEncryption = ({
     if (platform === "linux") {
       return (
         <>
-          For Ubuntu and Fedora Linux.
+          For Ubuntu, Kubuntu, and Fedora Linux.
```

Comment:
tbh since this is a corner case (tiny installed base compared to normal Ubuntu; should be able to confirm this with the customer that asked for this support), this feels like something we mention in docs and release notes and *don't* bother calling out in the UI, as for answering this particular question people are more likely to check (and search) docs than look at a tooltip.

Deferring to product on this though.